doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -209,71 +210,134 @@ infrastructure itself from the workloads compute resources.
+- **Other Features**: these additional features that are required by the
+networking solution as a whole, may be delivered by the **"Default CNI Plugin"**,
+or the **"CNI multiplexer/meta-plugin"** if it is deployed. For example,
+the integration of SDN solutions required by `req.inf.ntw.05` is enabled
+via CNI integration.
+- **Service Mesh**: The well known service meshes are "application service meshes"
+that address and interact with the application layer 7 protocols (eg.: HTTP)
+only. Therefore, their support is not required in this architecture, as these
+service meshes are outside the scope of the infrastructure layer of this
+architecture.
+
+<p align="center"><img src="../figures/ch03_networking.png" alt="Kubernetes Networking Architecture" Title="Kubernetes Networking Architecture" width="100%"/></p>
+<p align="center"><b>Figure 3-2:</b> Kubernetes Networking Architecture</p>
+
+<!--The above diagram is maintained here:
+https://wiki.lfnetworking.org/display/LN/CNTT+RA2+-+Kubernetes+-+Diagrams+-+Networking-->
+
+There are several types of low latency and high throughput networks required by
 telco workloads: signalling traffic workloads and user plane traffic workloads.
 Networks used for signalling traffic are more demanding than what a standard
 overlay network can handle, but still do not need the use of user space
 networking. Due to the nature of the signalling protocols used, these type of
-networks require NAT-less communication documented in infra.net.cfg.003 and will
+networks require NAT-less communication documented in `infra.net.cfg.003` and will
 need to be served by a CNI plugin with IPVLAN or MACVLAN support. On the other
 hand, the low latency, high throughput networks used for handling the user plane
 traffic require the capability to use a user space networking technology.
 
+| Requirement | Networking Solution with Multus | Networking Solution with DANM | Networking Solution with Tungsten Fabric | Networking Solution with NSM |
+|---|---|---|---|---|
+| The overlay network encapsulation protocol needs to enable ECMP in the underlay (`infra.net.cfg.002`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin | Supported | TBC |
+| NAT (`infra.net.cfg.003`) | Supported via the additional CNI plugin | Supported | Supported | TBC |
+| Network Policies (Security Groups) (`infra.net.cfg.004`) | Not supported | Not supported <sub>(1)<sub> | Supported | Supported via the default CNI plugin |
+| SFC support (`infra.net.cfg.005`) | Not relevant | Not relevant | Not relevant | Not relevant |
+| Traffic patterns symmetry (`infra.net.cfg.006`) | Not relevant | Not relevant | Not relevant | Not relevant |
+| Network resiliency (`req.inf.ntw.01`) | Supported | Supported | Supported | Supported |
+| Centrally administrated and configured (`req.inf.ntw.03`) | Supported via Kubernetes API | Supported via Kubernetes API | Supported via Kubernetes API | Supported via Kubernetes API |
```

Comment:
I think `req.inf.ntw.03` is about how the network resources (like IP address ranges) are configured and in this case Multus in itself does not support this. Others I guess support it via different CRD-s so I think it would be more precise to write "Suported via CRD-s".

Reply:
OK. I think to avoid misunderstanding the requirement should be clarified.  For example, in a Multus-based solution, I would manage all components through the k8s API - I would deploy and configure Multus via k8s API, configure physical devices using DPAPI, etc.  I have updated in https://github.com/cntt-n/CNTT/pull/2121/commits/1e963f31762bb213339d58ff765acaf87e6bd896 to say "Kubernetes API Server" as it is an API hosted on that control plane service.  Is that ok?

Reply:
Would a CNI that doesn't use CRDs be considered non-compliant? Most CNIs don't use CRDs to determine their IP, subnet, or routes. These are often configured in a config file. 

Furthermore, in NSM, these addresses may sometimes be assigned by dynamically with parameters chosen by the network service endpoint such as in the case of a VPN.

Reply:
@fkautz good points.  This seems related to @CsatariGergely comments here: https://github.com/cntt-n/CNTT/pull/2121#discussion_r545129024.

Would you both agree to the following:
- Default cluster network (and any other extensions): can be managed by config files, k8s API server (e.g. CRDs), or a combination.
- Additional network management plane: can be managed by config files, k8s API server (e.g. CRDs), or a combination.
- Additional network connections to Pods: must be managed by k8s API server (e.g. CRDs, DPAPI, etc.)
- Configuration of the additional network connections to Pods: can be k8s API server (e.g. CRDs) or an external management plane (e.g. dynamic address assignment)

Does that sound feasible?

Reply:
@fkautz @CsatariGergely are you ok with the wording I've added here: https://github.com/cntt-n/CNTT/commit/25235f3ffd99af431f0e94c8d8843d842bb7ab2b